docs/experimental/lakectl_local_posix.md

```diff
@@ -0,0 +1,99 @@
+---
+title: lakectl local file permissions tracking
+description: An experimental feature that allows tracking file ownership and permissions when using lakectl local
+parent: none
+---
+
+# lakectl local file permission tracking
+{: .d-inline-block }
+Experimental
+{: .label .label-green }
+
+This experimental feature is used to:
+* Support mode preservation for files and folders managed via lakectl local **for POSIX compliant filesystems only**
+* Support user and group preservation for files and folders managed via lakectl **local for POSIX compliant filesystems only**
+
+
+{: .note}
+> This feature is currently supported for Unix based Operating Systems only
+
+{: .warning }
+> Please make sure to contact the lakeFS team before enabling any experimental features!
+
+## Configuration 
+
+By default, this feature is disabled. Enabling file permission tracking is done via the configuration variable 
+`experimental.local.posix_permissions_enabled` which can be added to the lakectl.yaml file:
+
+   ```yaml
+   credentials:
+     access_key_id: AKIAIOSFDNN7EXAMPLEQ
+     secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+   metastore:
+     glue:
+       catalog_id: ""
+     hive:
+       db_location_uri: file:/user/hive/warehouse/
+       uri: ""
+   server:
+     endpoint_url: http://localhost:8000/api/v1
+   experimental:
+     local:
+       posix_permissions:
+         enabled: true
+   ```
+
+It is also possible to enable this feature via the corresponding environment variable:
+
+```
+LAKECTL_EXPERIMENTAL_LOCAL_UNIX_PERMISSIONS_ENABLED
+```
+
+## Usage
+
+lakectl local will automatically track changes in file permissions and ownership as part of the sync with the remote server.
+During the first sync with a remote path, lakectl local will need to update all the objects under this path in order to sync
+the permissions with lakeFS.
+When first creating the file and directory structure from an existing remote path the default permissions are set:
+- `0o0666-umask` mode for files
```

Comment:
1. The common way to write this (assuming new-style octal literals) is `0o0666 & ~umask`, and similarly for directories.
1. Consider noting that users with stricter umasks will change permissions on push.